js/helpers.js

```diff
@@ -41,7 +41,7 @@ function drawCalendar() {
     $.each(reqs, function(reqName, classData) {
         // set which class satisfied it
         var classSatisfier = '';
-        if (classData.satisfaction != 'notselected') {
+        if ($('#' + reqName + 'satisfaction')[0].length > 2 && classData.satisfaction !== 'notselected') {
```

Comment:
Logic I thought we already implemented but maybe it got refactored away -- don't display the satisfier if it's a 1:1 class (satisfier dropdown length < 3 -- the default select and the one class).